env.local

```diff
@@ -0,0 +1,6 @@
+NEXT_PUBLIC_FIREBASE_API_KEY=AIzaSyB-jj6Gvdn7F1TOhV4R84k2-RjIbjn5-F4
+NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN=cryptoapp-bab5b.firebaseapp.com
+NEXT_PUBLIC_FIREBASE_PROJECT_ID=cryptoapp-bab5b
+NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET=cryptoapp-bab5b.appspot.com
+NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=790877873296
+NEXT_PUBLIC_FIREBASE_APP_ID=1:790877873296:web:5c2bf5f8494b43ec18a415
```

Comment:
you pushed your secrets to github, you need to put your .env variables in your gitignore